### ISSUE TITLE
Cross compiling Veyon for Windows does not create installer on Ubuntu 16.04

### ISSUE THREAD

**Issue by jopsnow** [veyon/veyon#519]
**I tested this:**
sudo apt-get install python-software-properties -y
sudo add-apt-repository ppa:tobydox/mingw-w64 -y
sudo apt-get update

sudo apt install git g++ make cmake qtbase5-dev qtbase5-dev-tools qttools5-dev qttools5-dev-tools xorg-dev libxtst-dev libjpeg-dev zlib1g-dev libssl-dev libpam0g-dev libprocps-dev liblzo2-dev libqca2-dev libqca-qt5-2-dev libldap2-dev libsasl2-dev nsis tofrodos g++-mingw-w64 qt5base-mingw-w64 qt5tools-mingw-w64 qt5translations-mingw-w64 openssl-mingw-w64 libz-mingw-w64-dev libpng-mingw-w64 libjpeg-turbo-mingw-w64 interception-mingw-w64 openldap-mingw-w64 lzo2-mingw-w64 qca-mingw-w64 -y

sudo rm -R veyon* || true && git clone -b 4.1 --single-branch https://github.com/veyon/veyon.git && cd veyon && mkdir build && cd build && cmake -DCMAKE_INSTALL_PREFIX=/usr .. && make -j4 && fakeroot make package && sudo dpkg -i veyon_* && cp -R veyon_* ../../ && cd ../cmake && ./build_mingw64 && sudo make win-nsi && cp -R veyon-*setup* ../../ && cd ../../ && sudo rm -R veyon && git clone -b 4.1 --single-branch https://github.com/veyon/veyon.git && cd veyon && cd cmake && ./build_mingw32 && sudo make win-nsi && cp -R veyon-*setup* ../../ && cd ../../ &&  sudo rm -R veyon 

**then I got this error:** 
`CMake Error: File /home/jopper/veyon/3rdparty/kldap/src/kldap_config.h.cmake doe                     s not exist.
CMake Error at plugins/ldap/CMakeLists.txt:19 (CONFIGURE_FILE):
  configure_file Problem configuring file

-- CXX target ldap cotired.
-- CXX target authkeys cotired.

Veyon build summary
--------------------
* Version                     : 4.1.8.2 (4.1.8-2-g50430a9)
* Install prefix              : /usr
* Library & plugin directory  : /usr/lib/x86_64-linux-gnu/veyon
* Build type                  : relwithdebinfo
* Build platform              : x86_64
* Compile flags               :  -Wall -fstack-protector-strong  (CXX:  -Wall -f                     no-exceptions -std=c++11 -fstack-protector-strong )

-- Configuring incomplete, errors occurred!
See also "/home/jopper/veyon/build/CMakeFiles/CMakeOutput.log".
See also "/home/jopper/veyon/build/CMakeFiles/CMakeError.log".
`

**Comment by pauloboc** [comment]
Branch Master don't work!

Branch 4.1
Use: 
```
sudo apt-get install python-software-properties -y
sudo add-apt-repository ppa:tobydox/mingw-w64 -y
sudo apt-get update

sudo apt install git g++ make cmake qtbase5-dev qtbase5-dev-tools qttools5-dev qttools5-dev-tools xorg-dev libxtst-dev libjpeg-dev zlib1g-dev libssl-dev libpam0g-dev libprocps-dev liblzo2-dev libqca2-dev libqca-qt5-2-dev libldap2-dev libsasl2-dev nsis tofrodos g++-mingw-w64 qt5base-mingw-w64 qt5tools-mingw-w64 qt5translations-mingw-w64 openssl-mingw-w64 libz-mingw-w64-dev libpng-mingw-w64 libjpeg-turbo-mingw-w64 interception-mingw-w64 openldap-mingw-w64 lzo2-mingw-w64 qca-mingw-w64 cyrus-sasl-mingw-w64 -y


sudo rm -R veyon* || true && git clone --recursive -b 4.1 --single-branch https://github.com/veyon/veyon.git && sudo chmod -R 777 veyon/ && cd veyon && mkdir build && cd build && cmake -DCMAKE_INSTALL_PREFIX=/usr .. && make -j4 && fakeroot make package && sudo dpkg -i veyon_* && cp -R veyon_* ../../ && cd ../cmake && ./build_mingw64 && sudo make win-nsi && cp -R veyon-*setup* ../../ && cd ../../ && sudo rm -R veyon && git clone -b 4.1 --single-branch https://github.com/veyon/veyon.git && cd veyon && cd cmake && ./build_mingw32 && sudo make win-nsi && cp -R veyon-*setup* ../../ && cd ../../ &&  sudo rm -R veyon 
```

**Comment by jopsnow** [comment]
@pauloboc I tested your answer then I got these errors 
```
[ 65%] Building CXX object plugins/platform/windows/CMakeFiles/windows-platform.dir/moc_XEventLog.cpp
[ 65%] Linking CXX shared module windows-platform.dll
/usr/bin/x86_64-w64-mingw32-ld: cannot find -linterception
collect2: error: ld returned 1 exit status
plugins/platform/windows/CMakeFiles/windows-platform.dir/build.make:936: recipe for target 'plugins/p
make[6]: *** [plugins/platform/windows/windows-platform.dll] Error 1
CMakeFiles/Makefile2:1388: recipe for target 'plugins/platform/windows/CMakeFiles/windows-platform.di
make[5]: *** [plugins/platform/windows/CMakeFiles/windows-platform.dir/all] Error 2
Makefile:149: recipe for target 'all' failed
make[4]: *** [all] Error 2
CMakeFiles/win-nsi.dir/build.make:57: recipe for target 'CMakeFiles/win-nsi' failed
make[3]: *** [CMakeFiles/win-nsi] Error 2
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/win-nsi.dir/all' failed
make[2]: *** [CMakeFiles/win-nsi.dir/all] Error 2
CMakeFiles/Makefile2:74: recipe for target 'CMakeFiles/win-nsi.dir/rule' failed
make[1]: *** [CMakeFiles/win-nsi.dir/rule] Error 2
Makefile:184: recipe for target 'win-nsi' failed
make: *** [win-nsi] Error 2

```

**Comment by pauloboc** [comment]
Try sudo apt install cyrus-sasl-mingw-w64 -y

**Comment by jopsnow** [comment]
@pauloboc 
I tried: 
`sudo apt install cyrus-sasl-mingw-w64 -y`

then the output is: 
```
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package cyrus-sasl-mingw-w64
```

**Comment by pauloboc** [comment]
@tobydox 
Any changes in package installation?

**Comment by jopsnow** [comment]
[link](https://launchpad.net/~tobydox/+archive/ubuntu/mingw-w64?field.series_filter=xenial)

 cyrus-sasl-mingw-w64 is not available for ubuntu 16.04 xenial.
